types/plotly.js/index.d.ts

```diff
@@ -643,6 +643,21 @@ export interface PlotData {
 	xy: Float32Array;
 	error_x: ErrorBar;
 	error_y: ErrorBar;
+	/**
+	 * Array containing integer indices of selected points.
+	 * Has an effect only for traces that support selections.
+	 * Note that an empty array means an empty selection where the `unselected` are turned on for all points,
+	 * whereas, any other non-array values means no selection all where the `selected` and `unselected` styles have no effect.
+	 */
+	selectedpoints: number[];
```

Comment:
@peterblazejewicz At least `scatter3d` plot has no `selectedpoints` attribute in the data. Should `Partial` be used here too?

Reply:
The same with `histogram2dcontour`. I guess there are other plot types without `selectedpoints` in their data.

Reply:
let me revisit, thx for reviewing!